doc/whats_new/v1.0.rst

```diff
@@ -335,6 +335,11 @@ Changelog
   `sample_weight` passed by a user were overwritten during the fit.
   :pr:`20515` by :user:`Carsten Allefeld <allefeld>`.
 
+- |Feature| Added auto mode to :class:`feature_selection.SequentialFeatureSelection`.
+  If the argument `n_features_to_select` is `'auto'`, select features until
+  the score improvement does not exceed the argument `tol`.
```

Comment:
We should as well the deprecation cycle regarding the default value of `n_features_to_select`

Reply:
fix it with https://github.com/scikit-learn/scikit-learn/pull/20145/commits/f843d3772b846c8596462a8fbdc856c95f888109